src/assets/text/general.ts

```diff
@@ -38,7 +38,7 @@ export const general = {
   confirm: "Confirm",
   aleoWebsite: "https://aleo.org/",
   pageNotFound: "404:Page not found",
-  countdown: (val: string): string => `${val} left`,
+  countdown: (val: string): string => `${val}`,
```

Comment:
This looks useless... 
Am I missing something?

Reply:
I removed the "left" text from the timer. eg. "01:30 left", now it's just "01:30"